Conductor/Client/ApiClient.cs

```diff
@@ -200,12 +200,10 @@ private byte[] GetBytes(Stream input)
                         string refreshToken = configuration.GetRefreshToken();
                         headerParams["X-Authorization"] = refreshToken;
                         retryCount++;
+                        continue;
                     }
                 }
-                else
-                {
-                    break;
-                }
+                break;
```

Comment:
this effectively removes the while loop right? Or will it still do a retry to refresh the token until the max retry count is exhausted?

Reply:
@v1r3n It would retry on a expired token scenario; any other exception would exit to loop